kipoi/postprocessing/variant_effects/utils/seqplotting_deps.py

```diff
@@ -0,0 +1,353 @@
+import re
```

Comment:
put to `kipoi/external/concise` and split into two files. One for one-hot encoding and the other one for plotting. Also, copy over the unit-tests of concise https://github.com/gagneurlab/concise/blob/master/tests/preprocessing/test_sequence.py